api/config/notify.php

```diff
@@ -28,6 +28,8 @@
         'test_sms' => env('GCNOTIFY_TEMPLATE_TEST_SMS'),
         'test_bulk_email' => env('GCNOTIFY_TEMPLATE_TEST_BULK_EMAIL'),
         'test_bulk_sms' => env('GCNOTIFY_TEMPLATE_TEST_BULK_SMS'),
+        'application_deadline_approaching_en' => env('GCNOTIFY_TEMPLATE_APPLICATION_DEADLINE_APPROACHING_EN'),
```

Comment:
If we're going to be making two of these for every new notification this is going to get out of hand in a hurry.  I don't know what the solution is, though.